### ISSUE TITLE
Create Jumpscale-test actor to be used by ZeroCI

### ISSUE THREAD

**Issue by john-kheir** [threefoldtech/jumpscaleX_threebot#314]
This ticket will be part of that [story](https://github.com/threefoldtech/home/issues/420)

### Jumpscale-test actor should be doing the following:
- Uses scheduler  
- Deploys n numbers of containers based on number of test jobs that are defined in the yaml file
 example: yaml [file](https://github.com/threefoldtech/jumpscaleX_core/blob/development/zeroCI.yaml)
- Containers will be created from a constant stable flist, and can be changed later depending on the need 
-  Once the containers are ready, tests will be executed in parallel
- Aggregates the results from the containers to be displayed on the ZeroCI

**Comment by xmonader** [comment]
closing in favor of the new testing story.
